test/controllers/gems_controller_test.rb

```diff
@@ -15,4 +15,15 @@ def setup
       assert_response :success
     end
   end
+
+  test "should get show" do
+    VCR.use_cassette("response_info_gem") do
+      response = Net::HTTP.get_response(URI('https://rubygems.org/api/v1/gems/rails.json'))
+      assert_equal Hash, JSON.parse(response.body).class
+      get :show, id: 'rails'
+      assert_template :show
+      assert_select "title", "Information Gem | RubyGems AdoptionCenter"
+      assert_response :success
+    end
```

Comment:
@angela4713 you're getting the hag out of writing tests :smiley:.

Do you think this tests is really checking that this feature works as expected. I think the content in the page you're checking could probably be more specific to the feature you're building.